definitions/divorce/json/CaseEvent.json

```diff
@@ -1793,6 +1793,19 @@
     "PostConditionState": "*",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "07/05/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "updateBulkCaseHearingDetails",
+    "Name": "Update case with court hearing",
+    "Description": "Update case with court hearing details",
+    "DisplayOrder": 21,
+    "PreConditionState(s)": "AwaitingPronouncement",
+    "PostConditionState": "AwaitingPronouncement",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/generate-document?templateId=FL-DIV-GNO-ENG-00020.docx&documentType=coe&filename=certificateOfEntitlement",
```

Comment:
Does this exist?

Reply:
Yes, this has already been merged in: https://github.com/hmcts/div-case-orchestration-service/pull/269